isubit/luggage_events
# LUGG-659 Add alternate URL

Added _New and Improved_ alternate URL field. Note: these changes require the changes from the branch in this pull request or the location field label and description will be applied to the alternate URL field: https://github.com/isubit/luggage_events/pull/20
## Testing:
1. Pull down my changes.
2. Run drush fra -y and drush cc all
3. Go to the events view and verify that you can see the Alternate URL field in the listing of upcoming/archived events.
